Makefile

```
@@ -304,3 +304,8 @@ static-deps:
 	docker manifest push audius/eth-ganache:latest
 	@echo "Pushed audius/eth-ganache:latest"
 
+
```

add `.PHONY` because `push-buf` is not a real file
```suggestion
.PHONY: push-buf
```